codebuild_specs/publish_to_npm.yml

```diff
@@ -0,0 +1,11 @@
+version: 0.2
+env:
+  shell: bash
+  git-credential-helper: yes
```

Comment:
how does`git-credential-helper` work ?

Reply:
only the jobs that require .git credentials should use git-credential-helper; this injects the GH credentials via the git-credential-helper https://git-scm.com/docs/gitcredentials so that things like push work. Can only be used on jobs that are manually triggered via console/api.